Stylesheet/variable.css

```diff
@@ -72,4 +72,6 @@ body {
   --button-color-default: #1e3d58;
   --button-color-secondary: #bf9270;
   --text-bold: 600;
+  --card-padding: 0 0 0.5rem 0.4rem;
```

Comment:
You could use px unit for values <=10 px.

Reply:
Thanks I have actually removed that attribute will keep that in mind.